packages/composites/nexus-kiln/src/abi/FundValueCalculatorRouter.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "inputs": [{ "internalType": "address", "name": "_feeManager", "type": "address" }],
+    "stateMutability": "nonpayable",
+    "type": "constructor"
+  },
+  {
+    "inputs": [{ "internalType": "address", "name": "_vaultProxy", "type": "address" }],
+    "name": "calcGav",
+    "outputs": [
+      { "internalType": "address", "name": "denominationAsset_", "type": "address" },
+      { "internalType": "uint256", "name": "gav_", "type": "uint256" }
+    ],
+    "stateMutability": "view",
+    "note": "THIS HAS BEEN MODIFIED TO BE A VIEW FUNCTION",
```

Comment:
Is this note from the original ABI, or something we have changed?

Reply:
This is something we have changed

Reply:
Why did we need this change?
If it is just to be able to call this function without expecting state changes, can we use `callStatic`? https://docs.ethers.org/v5/api/contract/contract/#contract-callStatic

Reply:
Updated